gcloud_dataproc/load_dataset.py

```diff
@@ -216,6 +213,30 @@ def _create_es_nodes(settings, create_persistent_es_nodes=False):
     return ip_address
 
 
+def _get_es_node_settings(k8s_cluster_name, num_temp_loading_nodes):
+    return {
+        "DEPLOY_TO": k8s_cluster_name,
+        "CLUSTER_NAME": k8s_cluster_name,
+        "ES_CLUSTER_NAME": k8s_cluster_name,
+        "NAMESPACE": k8s_cluster_name,  # kubernetes namespace
+        "IMAGE_PULL_POLICY": "Always",
+
+        "CLUSTER_MACHINE_TYPE": "n1-highmem-4",
+        "ELASTICSEARCH_VERSION": "6.3.2",
+        "ELASTICSEARCH_JVM_MEMORY": "13g",
+        "ELASTICSEARCH_DISK_SIZE": "100Gi",
+        "ELASTICSEARCH_DISK_SNAPSHOTS": None,
+
+        "KIBANA_SERVICE_PORT": 5601,
+
+        "ES_NUM_PERSISTENT_NODES": num_persistent_nodes,
```

Comment:
would num_persistent_nodes be undefined?

Reply:
good catch